robottelo/ui/navigator.py

```diff
@@ -20,6 +20,7 @@ def menu_click(self, top_menu_locator, sub_menu_locator,
                 "arguments[0].click();",
                 tertiary_element,
             )
+        self.browser.refresh()
```

Comment:
Okay, that change will not affect `StaleElementException` as it seems it will happen very rarely, but all tests that has subtests are failing for me without that change. Probably, I need to investigate it more